test/types/type-provider.test-d.ts

```diff
@@ -469,7 +469,7 @@ expectAssignable(server.withTypeProvider<TypeBoxProvider>().get(
     res.send('hello')
     res.send(42)
     res.send({ error: 'error' })
-    expectType<(payload?: string | number | { error: string }) => typeof res>(res.code(200).send)
+    expectType<(payload?: string) => typeof res>(res.code(200).send)
```

Comment:
Can you please add more expectType assertions for the other status codes?

Reply:
done 👍